src/main/java/life/qbic/projectbrowser/components/LevelComponent.java

```diff
@@ -228,17 +230,21 @@ public void updateUI(final String type, final String id, final String filterFor)
 
                 samplesContainer.addBean(newBean);
 
-                ArrayList<ch.systemsx.cisd.openbis.generic.shared.api.v1.dto.DataSet> foundDataset =
-                    datasetFilter.get(sample.getIdentifier());
+                if(!isSampleTypeForProcessedData(sample.getSampleTypeCode())) {
```

Comment:
Would it not be more explicit to check if it's Raw data instead of checking if it's not processed data? 
```suggestion
                if(isSampleTypeForRawData(sample.getSampleTypeCode())) {
```